fluent-plugin-google-cloud.gemspec

```diff
@@ -23,7 +23,7 @@ eos
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
   gem.add_runtime_dependency 'google-api-client', '> 0.9'
   gem.add_runtime_dependency 'googleauth', '~> 0.4'
-  gem.add_runtime_dependency 'grpc', '= 1.0.1'
+  gem.add_runtime_dependency 'grpc', '~> 1.0', '< 1.0.2'
```

Comment:
I guess this is to make sure if any bug fixes are made to the `1.0.X` branch, we are able to get it? Just changed.